IdentityCore/src/telemetry/MSIDTelemetryEventStrings.m

```diff
@@ -134,4 +135,116 @@
 NSString *const MSID_TELEMETRY_VALUE_AUTHORITY_ADFS                  = @"adfs";
 NSString *const MSID_TELEMETRY_VALUE_AUTHORITY_B2C                  = @"b2c";
 
+#else // MSAL CPP
 
+// Telemetry event name
+NSString *const MSID_TELEMETRY_EVENT_DEFAULT_EVENT          = @"";
```

Comment:
Why do you have all those empty constants?

Reply:
They are also excluded, thanks!

Reply:
@fengga, I'm still seeing those empty constants. I'm unclear why those are needed.

Reply:
I think it's because of too much file changes and Github is not working correctly. I clicked the "File changed" button in this PR, navigated to MSIDTelemetryEventStrings.m, and found everything are excluded (no empty string definitions).